packages/common/types/passport.ts

```diff
@@ -0,0 +1,4 @@
+export type Passport = {
+  passport: string;
```

Comment:
It's a little confusing to have a type `Passport` with a property `passport`. Can we rename this something like `AccessTokens`?